src/client/js/formSummarization.js

```diff
@@ -0,0 +1,47 @@
+function handleSummarization(event) {
+    event.preventDefault();
+
+    // check what text was put into the form field
+    const url = document.getElementById('url').value;
+    const sentences = document.getElementById('sentences').value;
+    const resultContainer = document.querySelector('.summary.result');
+    const spinnerContainer = document.getElementById('spinner');
+    const language = document.getElementById('language');
+    const summary = document.getElementById('summary');
+
+    document.querySelectorAll('.result').forEach(it => {
+        it.style.display = 'none';
+    });
+    spinnerContainer.style.display = 'inherit';
+
+    console.log('::: Form Submitted :::');
+    fetch('http://localhost:8081/summarize', {
```

Comment:
Same as above. Not critical now, but not production ready.

Reply:
kk